ui/src/authentication/services/AuthenticationService.js

```diff
@@ -1,30 +1,77 @@
+import { http } from '@api';
+import { sessionManager } from './SessionManager';
+
 class AuthenticationService {
-  getTokenFromDOM = () => {
-    const element = document.getElementsByName('authorized_token');
-    if (element.length > 0 && element[0].hasAttribute('value')) {
-      return element[0].value;
+  loginWithOauthProvider = (nextUrl, providerUrl) => {
+    localStorage.setItem('ILS_USER_WAS_LOGGEDIN', true);
```

Comment:
@zzacharo `localStorage.setItem('ILS_USER_WAS_LOGGEDIN', true);` should be probably called on authentication success, not before starting the authentication... psw might be wrong or some other issue